test/misc.jl

```diff
@@ -55,3 +55,16 @@ end
         end
     end
 end
+
+@testset "binomlogpdf" begin
+    @testset "identity" begin
+        for p in [rand(50); 0; 1]
+            for n in 0:50
+                for k in [-50:50; randn(10)]
+                    # testing genetic Julia binompdf against Rmath version.
+                    @test binomlogpdf(n, Float32(p), k) ≈ log(binompdf(n, p, k))
```

Comment:
Tests fail as the tolerance is too small by default. I suspect this will actually work:
```suggestion
                    @test Float32(binomlogpdf(n, Float32(p), k)) == Float32(log(binompdf(n, p, k)))
```
Can you check that it's OK? Otherwise you can just specify a reasonable tolerance manually.

Reply:
Unfortunately this actually still fails.

however if I use `≈` instead of `==`, it works I think. Although the test dumps some odd output when `k` is not an integer.

Anyways, I made the appropriate change.